From 221f2b28402d761e13a7c26a5c3fc51080a1f346 Mon Sep 17 00:00:00 2001
From: Andrew Purtell <apurtell@apache.org>
Date: Fri, 4 Dec 2015 18:40:48 -0800
Subject: [PATCH 291/335] HBASE-14923 VerifyReplication should not mask the
 exception during result comparison (Vishal
 Khandelwal)

Reason: Bug
Author: Vishal Khandelwal
Ref: CDH-35763

Change-Id: I5aa445272b5a8c6ad7e6d12be6783c2e6d46f3cf
---
 .../mapreduce/replication/VerifyReplication.java   |    1 +
 1 file changed, 1 insertion(+)

diff --git a/hbase-server/src/main/java/org/apache/hadoop/hbase/mapreduce/replication/VerifyReplication.java b/hbase-server/src/main/java/org/apache/hadoop/hbase/mapreduce/replication/VerifyReplication.java
index f94dac9..60ac8ca 100644
--- a/hbase-server/src/main/java/org/apache/hadoop/hbase/mapreduce/replication/VerifyReplication.java
+++ b/hbase-server/src/main/java/org/apache/hadoop/hbase/mapreduce/replication/VerifyReplication.java
@@ -151,6 +151,7 @@ public class VerifyReplication extends Configured implements Tool {
             context.getCounter(Counters.GOODROWS).increment(1);
           } catch (Exception e) {
             logFailRowAndIncreaseCounter(context, Counters.CONTENT_DIFFERENT_ROWS, value);
+            LOG.error("Exception while comparing row : " + e);
           }
           currentCompareRowInPeerTable = replicatedScanner.next();
           break;
-- 
1.7.9.5

